### ISSUE TITLE
Error: No module named 'PyQt5.sip'

### ISSUE THREAD

**Issue by GiantRoot** [retext-project/retext#504]
I'm using python3.7 and pip 20.0.2
When I used pip to install ReText automatically and tried import ReText in python,
I always get this Error.
I find that's because pip automatically installed PyQt5.14 which doesnt well with sip.
I tried to install PyQt5.10 instead of 5.14, then the problem solved.
I just want to help people who have the same problem, so I submit this issue.
Hope this problem can be solved some how.

**Comment by mitya57** [comment]
Hi! Can you share the installation log of PyQt5 5.14?

I looked at the latest wheel file on PyPI and its metadata has:

    Requires-Dist: PyQt5-sip (>=12.7, <13)

So if pip was using the wheel, it should have also installed PyQt5-sip. Or pip was building from source? What is your OS?

**Comment by GiantRoot** [comment]
First, I was using Win10-64bit 1909. Maybe its the problem.
Second, I tried to get the installation log of PyQt5.14. Today (20200510), I uninstalled the retext and pyqt and reinstalled retext again. This time the Error didnt happen again...very confusing.
Sorry，I got confused by the installation now...and maybe cant help to solve the problem (which doesnt exist anymore).
But still, I think this issue may help some people at some points.
